ajax/libs/sinon.js/package.json

```diff
@@ -16,6 +16,15 @@
     "type": "git",
     "url": "https://github.com/sinonjs/sinon"
   },
+  "npmName": "sinon",
+  "npmFileMap": [
+    {
+      "basePath": "pkg",
+      "files": [
+        "**/+(sinon.|sinon-no-sourcemaps.)*"
```

Comment:
@extend1994 Sinon maintainer here. Regarding sinon-ie and other files that were in older versions, they haven't been produced for quite some time. At least the timers were no longer created as of sinon 1.17 (the last 1.x release), probably earlier. After 2.0 we produced only a couple of files.

If you have any questions regarding Sinon call out. I don't understand the discussion on basePath, though 😁